test/specs/routing.test.js

```diff
@@ -763,8 +773,9 @@ module.exports = fetchMock => {
 		});
 
 		describe('unmatched calls', () => {
-			it('throws if any calls unmatched', async () => {
+			it.only('throws if any calls unmatched', async () => {
```

Comment:
We should probably get rid of the `only` and console.log from here

Reply:
Good spot. I'd like to add better linting to this project to catch stuff like this